docs/integrations/sources/chameleon.md

```diff
@@ -0,0 +1,44 @@
+# Chameleon
+This page contains the setup guide and reference information for the [Chameleon](https://app.chameleon.io/) source connector.
+
+## Documentation reference:
+Visit `https://developers.chameleon.io/#/apis/overview` for API documentation
+
+## Authentication setup
+`Chameleon` uses API Key authentication,
+Refer `https://app.chameleon.io/settings/tokens` for getting your API key.
+
+
+## Configuration
+
+| Input | Type | Description | Default Value |
+|-------|------|-------------|---------------|
+| `api_key` | `string` | API Key.  |  |
+| `start_date` | `string` | Start date.  |  |
+| `limit` | `string` | Limit. Max records per page limit | 50 |
+| `filter` | `string` | Filter. Filter for using in the `segments_experiences` stream | tour |
+| `end_date` | `string` | End date.  |  |
+
+## Streams
+| Stream Name | Primary Key | Pagination | Supports Full Sync | Supports Incremental |
+|-------------|-------------|------------|---------------------|----------------------|
+| helpbars | id | No pagination | ✅ |  ✅  |
+| segments | id | No pagination | ✅ |  ✅  |
+| urls | id | No pagination | ✅ |  ✅  |
+| url_groups | id | No pagination | ✅ |  ✅  |
+| changes | id | No pagination | ✅ |  ❌  |
+| launchers | id | No pagination | ✅ |  ✅  |
+| tooltips | id | No pagination | ✅ |  ✅  |
+| tours | id | No pagination | ✅ |  ✅  |
+| surveys | id | No pagination | ✅ |  ✅  |
```

Comment:
@btkcodedev can you please add survey responses? https://developers.chameleon.io/#/apis/survey-responses

We (@nataliekwong and I and @lmossman and the whole team) are very interested in that connector ourselves and will definitely use it ;)